usaspending_api/etl/management/commands/update_missing_covid_awards.py

```diff
@@ -0,0 +1,195 @@
+import logging
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+logger = logging.getLogger("script")
+
+UPDATE_AWARDS_SQL = """
+WITH recent_covid_awards AS (
+    SELECT
+        DISTINCT ON
+        (faba.award_id) faba.award_id,
+        sa.is_final_balances_for_fy
+    FROM
+        financial_accounts_by_awards faba
+    INNER JOIN disaster_emergency_fund_code defc ON
+        defc.code = faba.disaster_emergency_fund_code
+        AND defc.group_name = 'covid_19'
+    INNER JOIN submission_attributes sa ON
+        sa.reporting_period_start >= '2020-04-01'
```

Comment:
Do we need to remember to change this each year?

Reply:
No, that is the beginning of valid monthly DABS submission periods